baybe/kernels/basic.py

```diff
@@ -43,3 +45,45 @@ class MaternKernel(Kernel):
 
     Only takes the values 0.5, 1.5 or 2.5. Larger values yield smoother interpolations.
     """
+
+    lengthscale_prior: Optional[Prior] = field(default=None)
+    """An optional prior on the kernel lengthscale."""
+
+    lengthscale_initial_value: Optional[float] = field(
+        default=None, converter=optional(float)
+    )
+    """An optional starting value for the kernel lengthscale."""
+
+    def to_gpytorch(self, *args, **kwargs):  # noqa: D102
+        # See base class.
+        from torch import Tensor
+
+        gpytorch_kernel = super().to_gpytorch(*args, **kwargs)
+        if self.lengthscale_initial_value is not None:
+            gpytorch_kernel.lengthscale = Tensor([self.lengthscale_initial_value])
```

Comment:
should become `torch.tensor(...)` with controlled `dtype`, I think (also for `ScaleKernel`)

Reply:
Changed.